### ISSUE TITLE
[Question] 

### ISSUE THREAD

**Issue by Decius07** [NathanCollaert/AugmentedHardcore#38]
Can you make that if a Player kills another Player that he gets 1 more Life?

**Comment by NathanCollaert** [comment]
Yes this is possible by using the life parts system and making it so if a player kills another player he/she gets enough life parts for another life. (This can all be done through the plugin)

**Comment by Decius07** [comment]
Ok tanks
